docs/deployment/getting-started-locally/manage-azurite-emulator.md

```diff
@@ -0,0 +1,57 @@
+# Manage Azure Blob Storage Emulator
+
+This document is a step-by-step guide on how to configure, deploy and cleanup [Azurite](https://github.com/Azure/Azurite#introduction), the `Azure Blob Storage` emulator, within a [kind](https://kind.sigs.k8s.io/) cluster. This setup is ideal for local development and testing.
+
+## 00-Prerequisites
+
+Ensure that you have setup the development environment as per the [documentation](../prepare-dev-environment.md).
+
+> **Note:** It is assumed that you have already created kind cluster and the `KUBECONFIG` is pointing to this Kubernetes cluster.
+
+### Installing Azure CLI
+
+To interact with `Azurite` you must also install the Azure CLI `(version >=2.55.0)`
+On macOS run:
+
+```bash
+> brew install azure-cli
+```
+
+For other OS, please check the [Azure CLI installation documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+## 01-Deploy Azurite
+
+```bash
+> make deploy-azurite
+```
+
+The above make target will deploy `Azure` emulator in the target Kubernetes cluster.
+
+## 02-Setup ABS Container
+
+We will be using the `azure-cli` to create a container. Export the connection string to enable `azure-cli` to connect to `Azurite` emulator.
```

Comment:
```suggestion
We will be using the `azure-cli` to create an [ABS container](https://azure.microsoft.com/en-us/products/storage/blobs). Export the connection string to enable `azure-cli` to connect to `Azurite` emulator.
```

Reply:
there is no need for this link as we are explicitly giving the azure-cli command to execute.